ssc/src/Pos/Ssc/Types.hs

```diff
@@ -154,6 +164,14 @@ data SscSecretStorage = SscSecretStorage
       sssEpoch      :: !EpochIndex
     } deriving (Generic, Show, Eq)
 
+deriveSimpleBi ''SscSecretStorage [
+    Cons 'SscSecretStorage [
+        Field [| sssCommitment :: SignedCommitment |],
+        Field [| sssOpening    :: Opening          |],
+        Field [| sssEpoch      :: EpochIndex       |]
+    ]]
+
+
```

Comment:
Maybe "unnecessary" was a bad word. I just meant inconsistent with the spacing throughout the module. I'm probably going to dismiss this review anyways because it doesn't even matter.

Reply:
Gotcha. I was wondering because I tend to align stuff like the above chunk (lining up all of the `|]`'s, which I think is what you mean. stylish-haskell will sometimes enforce indentation like that, although I'm not sure how we have it configured for cases like this ^